code/modules/mob/living/carbon/carbon.dm

```diff
@@ -226,6 +235,10 @@
 				thrown_thing = M
 				var/turf/start_T = get_turf(loc) //Get the start and target tile for the descriptors
 				var/turf/end_T = get_turf(target)
+
+				if(M.mob_size == MOB_SIZE_BIG) //Halve the throw distance if the target is big
+					throw_range_modifier *= 0.5
```

Comment:
With the beforementioned being as it is currently, throwing a big mob as a big mob will throw 0.75% (1*1.5 = 1.5 * 0.5 = 0.75)

Reply:
Yes.